### DISCUSSION TITLE
Diverse Feature Requests

### DISCUSSION THREAD

**Discussion by JetFlightGit** [iSoron/uhabits#2041] [Feature Requests]
Hey,

I have been using the App for a bit now and I really love the simplistic of the App and it does almost everything I like. Some of the things I noticed that might be nice being added I summarized here. Maybe those things were asked before or I have not found yet. But here I go:

1. Be able to group certain Habits like old my house hold stuff I would like in one group and so on. Work around is at the moment, use color and reorder the Tasks.
2. Make one time Tasks, I know it does not have anything directly with hab it's but doing your one time hab it's in time is a hab it itself. Creating Tasks for one time Events with due date would save me extra having to have a second app.
3. Not one time task but I also like to add task without a due date. I just want to keep track how often I have to use the Washington Maschine. Dont really have to do that ones a week but just would like to check how often I do use it
4. Would be nice to have an advanced yes / no.for keep track of your mood for example I can manual write 1 through 5 but would be nice if there is an option already so I can just choose between 1-5, maybe even just choose from some small smilie. Super good neutral bad worse...
5. I like the option that you can quickly tab to get Checkmark / remove it. If I also use the option with spaces and question Marks I need to cycle through a lot. Would be nice if there an option to only cycle between check Mark and no Checkmark and if you want the other options you need to long press.
6. Highlight a suggested day, atm I need to feed my sourdough starter every week. I normaly do that on Friday. It's really nice that all of the other Checkmark are Show so I know when it's time again. If I feed the starter one day late everything get shifted. I would prefer to feed on Friday again though even though it's only 6 days then. So Highlighting the suggested day would be nice, and also dont hide that task even though a "uncolored Checkmark" ich is place so I dont miss the task just because it's hidded. Does that makes sense?

Sry for the long test but hopefully this stuff makes sort of sense. The biggest issue i am having is point 5.i like hiding my Tasks im done with but everything gets hide even though I have prefered day and would like to get into that rythem and again and not get pushed back one day because I dont see the task.

Best regards,
Dennis

**Comment by KyleSCraig** [comment]
For #6 I recommend ticking it in the past instead. I have some habits I need to wait for a certain day of the month to do. For example one is on the first of the month but I did it two days late this month. I just ticked it for the first. It doesn't give me data about when I have actually done it true, but it does mean it comes off hidden on the first of the next month.

Having every week treated differently from every 7 days is something many people have brought up and hopefully can be improved.

**Comment by ezio-melotti** [comment]
1. You can find more discussions about grouping here:
   * https://github.com/iSoron/uhabits/discussions/495
2. I ended up using [Tasks](https://tasks.org/).  I don't think one-off todos belong with habits, and Tasks already have the features you asked and more (e.g. setting start/due date).
3. A workaround could be setting a habit with a low frequency (e.g. once a month), and then mark whenever you do it.
4. This sounds similar to this:
   * https://github.com/iSoron/uhabits/discussions/979
5. From the widgets it cycles through `?`/`v`/`-`/`x` with each tap.  From the main screen a tap brings up a popup that gives you the four options, whereas a long press cycles through the above list.  Most of the times I just single tap on the widget to go from `?` to `v`.
6. For this I agree with @KyleSCraig and use the same strategy.
